coins/coins.go

```diff
@@ -180,7 +180,7 @@ func (a *WeiAmount) BigInt() *big.Int {
 	}
 	if cond.Rounded() {
 		// We round when converting from Ether to Wei, so we shouldn't see this
-		log.Warn("Converting WeiAmount=%s to big.Int required rounding", a)
+		log.Warn("converting WeiAmount=%s to big.Int required rounding", a.String())
```

Comment:
I thought we were using the convention that error messages start with a lower case letter and log messages start with an upper case letter. `String()` is called automatically when using `%s`.

Reply:
Whatever we decide, we should probably harmonize all the logs to use the same standard. I'd been using first letter capitalized in log messages due to previous places of employment requiring it.

Reply:
ah I need to change this to `Warnf` lol

I'm used to not capitalizing log messages, and I agree we should pick one and stick with it (I vote for lowercase) but as long as it's consistent that's what's important